### ISSUE TITLE
After the day ended you are repositioned inside the room

### ISSUE THREAD

**Issue by marCOmics** [Spexify/emergenCity#72]

**Comment by cccynic** [comment]
That is a feature, I can remove it if it bothers you - the idea was that when you end the day at another scene or go to "sleep" at home, you get transported back home at the position.

**Comment by cccynic** [comment]
see #60

**Comment by marCOmics** [comment]
Yeah, I noticed it later, I didn't know about this, but it makes sense, so it's good that you did it! xD
